src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -384,6 +384,7 @@ private FindSeriesResult FindSeries(ParsedEpisodeInfo parsedEpisodeInfo, int tvd
                     return new FindSeriesResult(searchCriteria.Series, SeriesMatchType.Title);
                 }
 
+                _logger.Debug("Could not match Searched Series [{0}] to Parsed Series [{1}], attempting id match", searchCriteria.Series.CleanTitle, parsedEpisodeInfo.SeriesTitle);
```

Comment:
`CleanTitle` is not the only title searched for, nor is it the only title used when parsing, so I don't think this makes things any clearer.

Besides not really adding clarity, I don't think a second log message to convey this is necessary and the mix of title case doesn't fit.